testing/trino-product-tests/src/main/java/io/trino/tests/product/hive/TestHiveTransactionalTable.java

```diff
@@ -1836,6 +1837,33 @@ public void testDoubleUpdateAndThenReadFromHive()
         });
     }
 
+    @Test(groups = HIVE_TRANSACTIONAL)
+    public void testDeleteWithOriginalFiles()
+    {
+        withTemporaryTable("test_delete_with_no_predicate_on_ctas", true, false, NONE, tableName -> {
+            onTrino().executeQuery("SET SESSION scale_writers = true"); //these 3 properties are necessary to make sure there is more than 1 original file created
```

Comment:
don't use end-of-line comment for a thing that spans more than one line.
put the comment before the lines

```suggestion
            // these 3 properties are necessary to make sure there is more than 1 original file created
            onTrino().executeQuery("SET SESSION scale_writers = true"); 
```